torch/_inductor/codegen/triton_utils.py

```diff
@@ -0,0 +1,315 @@
+import dataclasses
```

Comment:
is this file just the split of original triton.py, or is there something to review here?

Reply:
Yeah it's a spli